app/pages/sections/catalogue-solutions/order-item/contextCreator.test.js

```diff
@@ -130,4 +138,77 @@ describe('catalogue-solutions order-item contextCreator', () => {
       expect(context.saveButtonText).toEqual(manifest.saveButtonText);
     });
   });
+  describe('getErrorContext', () => {
+    const mockValidationErrors = [{
+      field: 'quantity',
+      id: 'quantityRequired',
+    },
+    {
+      field: 'price',
+      id: 'priceRequired',
+    }];
```

Comment:
a blank line after this please. also the contents of these objects can be inlined if you want to save on some space
```
{ field: 'quantity', id: 'quantityRequired' }
```